plugins/woocommerce-blocks/assets/css/editor.scss

```diff
@@ -28,11 +28,11 @@
 
 // Style inline notices in the inspector.
 .components-base-control {
-	+ .wc-block-base-control-notice {
+	+.wc-block-base-control-notice {
 		margin: -$gap 0 $gap;
 	}
 
-	+ .wc-block-base-control-notice:last-child {
+	+.wc-block-base-control-notice:last-child {
```

Comment:
> Is this change just cosmetic or it has some implications? AFAIK it would compile to the same CSS so I'm curious.

It's just cosmetic. When I saved the file, it automatically formatted the CSS. I thought I could keep the auto-formatted code. Is it auto-formatting to another format when you save the file? I think auto-formatting is following the `.markdownlint.json` file. 🤔

Reply:
I have reverted this change in e9a1151.